legal-api/src/legal_api/resources/business/business_tasks.py

```diff
@@ -39,11 +41,31 @@ class TaskListResource(Resource):
     def get(identifier):
         """Return a JSON object with meta information about the Service."""
         business = Business.find_by_identifier(identifier)
+        is_nr = identifier.startswith('NR')
```

Comment:
curious, could we use the namex service here instead to determine if this is a business coming from NR? I don't think we can guarantee new companies' identifiers will start with 'NR' (I know we have this validation elsewhere, so maybe we keep it this way for now?)

Reply:
I think this will change once we solidify how we are handling identifiers, but currently the UI feeds in NR prefixed string. So we will probably address this in a future PR.